src/enkf/gridio_fv3reg.f90

```diff
@@ -518,81 +519,86 @@ subroutine writegriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,vargrid
 
        deallocate(vworkvar3d)
     endif
+
+     if(q_ind>0) then
+    
+       varstrname='sphum'
+        call read_fv3_restart_data3d(varstrname,fv3filename,file_id,qbgworkvar3d)
+          do k=1,nlevs
+              nn = nn_tile0
+          do j=1,ny_res
+             do i=1,nx_res
+                nn=nn+1
+                workinc3d(i,j,k)=vargrid(nn,levels(q_ind-1)+k,nb,ne) 
+             enddo
+          enddo
+          enddo
+          qworkvar3d=qbgworkvar3d+workinc3d   
+     
+       call write_fv3_restart_data3d(varstrname,fv3filename,file_id,qworkvar3d)
+       do k=1,nlevs
+          if (nproc .eq. 0)                                               &
+             write(6,*) 'WRITEregional : sphum ',                           &
+                 & k, minval(qworkvar3d(:,:,k)), maxval(qworkvar3d(:,:,k))
+       enddo
+    end if
+
+
+
```

Comment:
Please help to fix the indentation in this code section: lines 522-547 if possible.